### PR TITLE
Docs: Add explanation about little/big endian

### PR DESCRIPTION
**Pull request by Granjow**
Aims to help users like me which always have to look up which way round endianness now works.

----
:books: Documentation preview :books:: https://cpython-previews--109841.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/struct.rst [hunk 1]
```diff
@@ -160,6 +160,11 @@ following table:
 
 If the first character is not one of these, ``'@'`` is assumed.
 
+The number ``1023``, or ``0x3ff`` in hex, is represented as follows in bytes:
+
+* ``0x03 0xff`` in big-endian (``>``)
+* ``0xff 0x03`` in little-endian (``<``)
```
Comment (AA-Turner) [inline Doc/library/struct.rst]: You could make this a `.. tip` or hint?

File: Doc/library/struct.rst [hunk 2]
```diff
@@ -160,6 +160,11 @@ following table:
 
 If the first character is not one of these, ``'@'`` is assumed.
 
+The number ``1023``, or ``0x3ff`` in hex, is represented as follows in bytes:
```
Comment (AA-Turner) [inline Doc/library/struct.rst]:
```suggestion
The number 1023 (``0x3ff`` in hexadecimal) has the following byte representations:
```

**Review by AA-Turner — COMMENTED** [review]
You could maybe write the byte pairs as Python byte strings. But that has cons. Also consider extracting this note into an admonition like `tip` or `hint` or `info`.

A

**Review by Granjow — COMMENTED** [review]
Sorry, this MR drowned in my email inbox. Better now? @AA-Turner

**Review by hauntsaninja — COMMENTED** [review]
Thanks for the PR! It looks like the doctest is failing, maybe add the import?

**Review by kushaldas — APPROVED** [review]
CI is good now.

**Comment by Granjow** [comment]
Wow, that went fast now, thanks a lot!

**Pull request merged**